examples/protocol_specification_ex/fipa.yaml

```diff
@@ -32,3 +32,14 @@ ct:Query: |
 ct:Description: |
   bytes description = 1;
 ...
+---
+reply:
+  cfp: [propose, decline]
+  propose: [accept, accept_w_inform, decline]
+  accept: [decline, match_accept, match_accept_w_inform]
```

Comment:
should maybe add `inform` as a valid reply, same for match_accept

Reply:
Done